README.md

```diff
@@ -24,25 +24,49 @@ EluvioLive CLI
 Usage: elv-live <command>
 
 Commands:
-  nft_add_contract <library> <object>       Add a new or existing NFT contract
-  <tenant> [minthelper] [cap] [name]        to an NFT Template object
-  [symbol] [nftaddr] [hold]
-  nft_add_minter <addr> <minter>            Add a new or existing NFT contract
-                                            to an NFT Template object
+  nft_add_contract <tenant> <object> <cap>  Add a new or existing NFT contract
+  <name> <symbol> [options]                 to an NFT Template object
+  token_add_minter <addr> <minter>          Add minter or mint helper address to
+                                            NFT or Token
+  token_renounce_minter <addr>              Renounce the minter(msg.sender) from
+                                            NFT or Token
+  token_is_minter <addr> <minter>           check if minter to NFT or Token
   nft_set_proxy <addr> [proxy_addr]         Set a proxy on an NFT contract
   nft_balance_of <addr> <owner>             Call NFT ownerOf - determine if this
                                             is an owner
-  nft_show <addr>                           Show info on this NFT
+  nft_show <addr> [options]                 Show info on this NFT
+  nft_transfer <addr> <token_id> <to_addr>  Transfer the specified NFT as the
+  [options]                                 token owner
   nft_proxy_transfer <addr> <token_id>      Tranfer NFT as a proxy owner
   <from_addr> <to_addr>
   nft_build <library> <object> [options]    Build the public/nft section based
                                             on asset metadata. If --nft_dir is
                                             specified, will build a generative
                                             nft based on *.json files inside the
                                             dir. See README.md for more details.
-  nft_lookup <addr> <token_id>              Decode and look up a local NFT by
+  nft_burn <addr> <token_id>                Burn the specified NFT as the owner
```

Comment:
were these all just missing?

Reply:
yes, I think so.

I ran --help and cut/paste the output to README for all the new stuff.  The diff might have some things out of order.